content/en/docs/reference/glossary/cgroups.md

```diff
@@ -4,7 +4,7 @@ id: cgroup
 date: 2019-06-25
 full_link:
 short_description: >
-  cgroup (abbreviated from control group) is a Linux kernel feature that limits, accounts for, and isolates the resource usage (CPU, memory, disk I/O, network) for a collection of processes.
+  A group of Linux processes with optional resource isolation, accounting and limits.
```

Comment:
- Question: Should the file be renamed to `cgroup.md`?
- Is the short description equivalent to this statement from http://man7.org/linux/man-pages/man7/cgroups.7.html:
A cgroup is a collection of processes that are bound to a set of limits or parameters defined via the cgroup filesystem.
- nit picky: Why is `optional` part of the sentence?

Reply:
Creating a cgroup and imposing a limit are separate operations, so I think it's fine to point out that limits are optional.
Not all cgroup types have limits, eg [freezer](https://www.kernel.org/doc/Documentation/cgroup-v1/freezer-subsystem.txt) doesn't.

Reply:
OK